akka-persistence/src/test/scala/akka/persistence/serialization/SerializerSpec.scala

```diff
@@ -221,6 +221,31 @@ class MessageSerializerPersistenceSpec extends AkkaSpec(customSerializers) {
       }
     }
 
+    "given PersistentRepr serialized with Akka 2.3.11 Scala 2.10" must {
```

Comment:
I actually did that on purpose, to make sure that we can replay these things when they are serialized with 2.10 - 2.3.11